docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,193 @@
+# Using the Network Shim
+
+## About
+
+The Network Shim is a tool that provides a cypress plugin as well as a setup utilility for cypress' `setupFile`, which have to be used in conjunction. It uses the cypress `cy.intercept()` API to intercept network traffic during a test run. It has three different _modes_:
```

Comment:
Maybe you could start with a succinct explanation of what the network shim is meant to be used for. Like:

> The network shim allows you to record and play back network traffic during your tests. This means you don't have to make live network requests when running e2e tests on CI, which is generally considered an anti-pattern as it can lead to flaky tests.

Reply:
I think I'd put the succinct explanation completely at the top. Then followed by an install command (if that's necessary). And that followed by a very short TLDR that lists the simplest, shortest way to get started. And then after that we can go more in depth.

That to me often is the most readable manner to document. Basically going from simple/high level -> complex/detailed.

Reply:
I think this is a good suggestion but I don't think I can implement it exactly....
1. Is sort of done already by tweaking the about section
2. Doesn't completely exists, but the `cli-utils-cypress` CLI does indeed come with an install command, which should generate (most of) the required files.
3. I see 2 & 3 being overlapping in our case, I'll probably end up combining these
4. I think I've got the in-depth part covered 🙄 
5. I will just create some clickable list leading to the headers

I'll ping you when I'm done